src/v/cluster/feature_table.cc

```diff
@@ -64,7 +66,7 @@ std::string_view to_string_view(feature_state::state s) {
 
 // The version that this redpanda node will report: increment this
 // on protocol changes to raft0 structures, like adding new services.
-static constexpr cluster_version latest_version = cluster_version{6};
+static constexpr cluster_version latest_version = cluster_version{7};
```

Comment:
i don't think you need to bump this version because it was already bumped for v22.3 here:

```
  feature_spec{
    cluster_version{6},
    "transaction_ga",
    feature::transaction_ga,
    feature_spec::available_policy::always,
    feature_spec::prepare_policy::always},
```

see the wiki page on the feature manager for more details (i'd add a link here but its the private wiki)

Reply:
You're right. I looked at the commit date and the change was committed before the release, but after v22.2 was cut. Will remove.